code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -13,7 +13,8 @@
 #define NITRYL_FORMATION_ENERGY				100000
 #define TRITIUM_BURN_OXY_FACTOR				100
 #define TRITIUM_BURN_TRIT_FACTOR			10
-#define TRITIUM_BURN_RADIOACTIVITY_FACTOR	1000000 //The neutrons gotta go somewhere. Completely arbitrary number.
+#define TRITIUM_BURN_RADIOACTIVITY_FACTOR	50000 //The neutrons gotta go somewhere. Completely arbitrary number.
```

Comment:
this is going from 1 million to 50k, it's quite huge of a change

Reply:
the rad proc is now rad(STUFF && prob(10)) so every burst needs to produce 10 times as much energy to compensate, 1m/50k=20 so in effect I just doubled it. That still might be a lot, but I'm serious when I said I didn't balance the radiation it makes a lot; I was just changing it up/down by factors of 10.

Actually, i probably should've left it one order of magnitude higher. When I was testing it I put trit+ox cans in the atmos mix chamber and ignited them while standing in them, to see how many rads I got. Tritum is *supposed* to produce enough radiation while burning to produce useful amounts of power, so in a ideal situation like that you'd probably expect to get about as many rads as sitting next to the SM for a good while. I'll do some more testing later.

Reply:
I think, you should get more than gives SM.Because tritium is expends,SM not

Reply:
Exactly. By calibrating it with a room of high-o2 tritium burnmix I probably set the radiation too low.